### DISCUSSION TITLE
Posting and liking tweets.

### DISCUSSION THREAD

**Discussion by Reporter1** [j-fbriere/squawker#301] [Q&A]
Hi I just started using squawker, I am logged in with a twitter account.
I think the answer is no but I want to ask can you post / reply / like (heart button) posts?

I can't seem to reply or like / heart posts.

Is squawker a read / view only app? Apart from downloading media.

Thanks

**Comment by Reporter1** [comment]
I'm guessing it's read only? As in can't reply or like tweats?

**Comment by j-fbriere** [comment]
Indeed it is read only.

**Comment by szamanr** [comment]
is writing and liking possible and on the roadmap, or is it not possible or not desired to implement?
